core/src/main/java/org/apache/accumulo/core/file/blockfile/cache/impl/BlockCacheConfiguration.java

```diff
@@ -41,13 +42,16 @@ public class BlockCacheConfiguration implements Configuration {
 
   private final long summaryMaxSize;
 
-  public BlockCacheConfiguration(AccumuloConfiguration conf) {
-    genProps = conf.getAllPropertiesWithPrefix(Property.TSERV_PREFIX);
+  public BlockCacheConfiguration(AccumuloConfiguration conf, Property serverPrefix,
```

Comment:
Just an idea, but the code could be a little less susceptible  to coding error if we pass in a single flag for TSERVER vs SSERVER.  The properties used for each could be encoded in this class.  This is just a thought and does not require change unless you agree with my argument.